src/components/external-provider-login.tsx

```diff
@@ -2,6 +2,9 @@ import { Provider } from '@supabase/supabase-js';
 import { Capacitor } from '@capacitor/core';
 import { supabase } from '~/services/supabase-client';
 import { Button } from '~/components/button';
+import { GoogleIcon } from './icons/google-icon';
+import { FacebookIcon } from './icons/facebook-icon';
+import { AppleIcon } from './icons/apple-icon';
```

Comment:
I think the icons are missing or the paths are wrong.

Reply:
🤦🏼‍♂️

Reply:
I've added it